app/javascript/components/helpers/helpers.js

```diff
@@ -1,7 +1,7 @@
 export const isEmpty = (obj) => Object.keys(obj).length === 0 && obj.constructor === Object
 
 export function isBlank(obj) {
-  return typeof obj === 'undefined' || obj === null;
+  return typeof obj === 'undefined' || obj === null || obj === 0;
```

Comment:
@LyubovP I think it is not good idea to add condition `obj === 0` here, this method should return true only if object is 'undefined' or 'null'.

Reply:
@SBorozenets If I remove || obj === 0 from the isBlank function, then the function will return true for the following values: undefined, null. In our case, the productivity value is equal to 0, which is not included in the list of values that return true from the isBlank function. Therefore, if I remove || obj === 0, the "Next" button will be active even for productivity values ​​that are equal to 0. (

Reply:
In that case maybe rename the method from `isBlank` to `notAnswered`?

Reply:
@LyubovP just add condition in `disabled`:
 disabled={isBlank(productivity) || productivity === 0}